packages/www/app/routes/resume.tsx

```
@@ -0,0 +1,275 @@
+										return (
+											<li key={link.text}>
+												<a
+													className="flex items-center space-x-2 text-blue-800 underline md:justify-end"
+													href={link.href}
+													aria-label={`Visit ${link.text}`}
+												>
+													<span className="print:hidden">{link.text}</span>
+													<span className="hidden print:inline">
+														{url.pathname}
+													</span>
+													<svg className="h-4 w-4 text-black" aria-hidden>
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)